src/main/java/htsjdk/samtools/util/Histogram.java

```diff
@@ -42,40 +35,46 @@
  *
  * @author Tim Fennell
  */
-public class Histogram<K extends Comparable> extends TreeMap<K, Bin> {
+public final class Histogram<K extends Comparable<K>> implements Serializable {
```

Comment:
I agree with @tfenne - you may want a histogram of things that are not serializable. Look at all java collections - none specifies keys/values to be serializable.